src/Manifolds.jl

```diff
@@ -636,12 +637,14 @@ export AbstractCotangentVector,
     AbstractManifold, AbstractManifoldPoint, AbstractTangentVector
 # Manifolds
 export AbstractSphere, AbstractProjectiveSpace
+export MatrixManifold
```

Comment:
```suggestion
```

Reply:
oops, missed that. Surprised Aqua did not complain that we export a non defined thingy.